### PR TITLE
Indicate grouping for code inspections when grouped by location

### LINKED ISSUE #2777: Code Inspections: Grouping By Location - State what source is
Code Inspection doesn't have any identifier for group by location where it tells you if it's a Module, Workbook, Worksheet, etc...

### PR DESCRIPTION
**Pull request by IvenBach**
Close #2777.

### CODE REVIEW

File: Rubberduck.VBEEditor/QualifiedSelection.cs [hunk 1]
```diff
@@ -12,6 +13,8 @@ public QualifiedSelection(QualifiedModuleName qualifiedName, Selection selection
 
         public QualifiedModuleName QualifiedName { get; }
 
+        public string QualifiedNameAndComponentType => $"{QualifiedName} - {QualifiedName.ComponentType}";
```
Comment (MDoerner) [inline Rubberduck.VBEEditor/QualifiedSelection.cs]: This does not belong here.

This is clearly a display concern, which does not belong here or on the QMN itself. This should be handled in some converter in the code responsible for displaying the information.

File: Rubberduck.Core/UI/Controls/GroupingGrid.xaml [hunk 2]
```diff
@@ -1,136 +1,136 @@
-﻿<DataGrid
```
[the remark this answers is not in the document]
Comment (IvenBach) [reply Rubberduck.Core/UI/Controls/GroupingGrid.xaml]: There shouldn't be anything in this file that changed. Best I recall I tried changing a couple things but ultimately reverted them.

File: Rubberduck.VBEEditor/QualifiedSelection.cs [hunk 3]
```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using Rubberduck.VBEditor.SafeComWrappers;
```
[the remark this answers is not in the document]
Comment (IvenBach) [reply Rubberduck.VBEEditor/QualifiedSelection.cs]: Added without me catching it.

File: Rubberduck.Core/UI/Controls/GroupingGrid.xaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (bclothier) [reply Rubberduck.Core/UI/Controls/GroupingGrid.xaml]: It seems to be whitespace only changes, so say GitHub.

**Comment by retailcoder** [comment]
This diff is impossible to review... I wish I could ignore this and merge it anyway, but we.. can't have that. Please submit the changes in a new PR from a clean branch.

**Pull request closed without merging**